askama_derive/src/generator.rs

```diff
@@ -447,6 +447,25 @@ impl<'a> Generator<'a> {
         ))?;
         buf.writeln("::askama_hyper::respond(value)")?;
         buf.writeln("}")?;
+        buf.writeln("}")?;
+        buf.writeln(&format!(
```

Comment:
At least an empty line here would be nice, maybe add a coment here and before the other impl writing, too?